controllers/controllers.go

```diff
@@ -0,0 +1,21 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/app"
+	"github.com/Diegoplas/go-bootcamp-deliverable/useCase"
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
+)
+
+func GetPokemon(w http.ResponseWriter, r *http.Request) {
+
+	requestIndex := mux.Vars(r)["pokemon_id"]
+
+	wantedIndex := app.ValidateID(requestIndex)
```

Comment:
I'm guessing that after making some changes this function will return an error, it's important to handle it and based on what the error was we could return to the user an appropriate status code. For example in this case, if the id is not valid we could return a BadRequest status. 

Also remember that we should return exactly what the error was. We should return a message that won't expose the internal functionality of the API, like `invalid id param` or something along those lines